ui/pages/create-account/institutional/connect-custody/account-list.js

```diff
@@ -0,0 +1,246 @@
+import PropTypes from 'prop-types';
+import React, { useState, useEffect, useRef } from 'react';
+import copy from 'copy-to-clipboard';
+import Button from '../../../../components/ui/button';
+import CustodyLabels from '../../../../components/institutional/custody-labels';
+import { SWAPS_CHAINID_DEFAULT_BLOCK_EXPLORER_URL_MAP } from '../../../../../shared/constants/swaps';
+import { CHAIN_IDS } from '../../../../../shared/constants/network';
+import { SECOND } from '../../../../../shared/constants/time';
+import { shortenAddress } from '../../../../helpers/utils/util';
+import Tooltip from '../../../../components/ui/tooltip';
+import {
+  Color,
+  TextVariant,
+  JustifyContent,
+  BLOCK_SIZES,
+  DISPLAY,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import Box from '../../../../components/ui/box';
+import {
+  Text,
+  Label,
+  Icon,
+  ICON_NAMES,
+  ICON_SIZES,
+  ButtonLink,
+} from '../../../../components/component-library';
+
+export default function CustodyAccountList({
+  rawList,
+  accounts,
+  onAccountChange,
+  selectedAccounts,
+  onCancel,
+  onAddAccounts,
+  custody,
+}) {
+  const timerRef = useRef(null);
+  const t = useI18nContext();
+  const [copied, setCopied] = useState(false);
+
+  useEffect(() => {
+    return () => clearTimeout(timerRef.current);
+  }, []);
+
+  const getButtonLinkHref = (account) => {
+    const url = SWAPS_CHAINID_DEFAULT_BLOCK_EXPLORER_URL_MAP[CHAIN_IDS.MAINNET];
+    return `${url}address/${account.address}`;
+  };
```

Comment:
This function does not need to be in the body of this component but can be declared outside of it to avoid re-allocating the function eveyrtime the component re-renders.

Reply:
Fixed